tests/CacheTest.php

```diff
@@ -20,92 +22,131 @@ public function setUp()
         $this->registrar->forgetCachedPermissions();
 
         DB::connection()->enableQueryLog();
+    }
 
-        $this->assertCount(0, DB::getQueryLog());
+    /** @test */
+    public function it_can_cache_the_permissions()
+    {
+        $this->registrar->getPermissions();
 
-        $this->registrar->registerPermissions();
+        $this->assertQueryCount(self::QUERIES_PER_CACHE_PROVISION);
 
-        $this->assertCount(2, DB::getQueryLog());
+        $this->registrar->getPermissions();
 
-        DB::flushQueryLog();
+        $this->assertQueryCount(self::QUERIES_PER_CACHE_PROVISION);
```

Comment:
If it's been cached, shouldn't the queryCount be 0?

Reply:
Sigh - I missed that you didn't do a "reset" in between, unlike the other tests. :)
So, this is correct: should return the same as it did 2 statements earlier.
👍